core/src/main/java/tachyon/Constants.java

```diff
@@ -59,6 +59,9 @@
   public static final int DEFAULT_WORKER_MAX_WORKER_THREADS = 2048;
 
   public static final int DEFAULT_USER_FAILED_SPACE_REQUEST_LIMITS = 3;
+  
+  public static final boolean DEFAULT_USER_DISABLE_LOCAL_READ = false;
```

Comment:
I see.
My original thinking is that local read should be opt in for most cases (at
least these days). Using "ENABLE" makes it like its an
optimization/feature.
But agreed that consistency is maybe more important.

On Thu, Apr 30, 2015 at 12:44 PM, Bin Fan <notifications@github.com> wrote:

> In core/src/main/java/tachyon/Constants.java
> <https://github.com/amplab/tachyon/pull/868#discussion_r29463025>:
>
> > @@ -59,6 +59,9 @@
> >    public static final int DEFAULT_WORKER_MAX_WORKER_THREADS = 2048;
> >
> >    public static final int DEFAULT_USER_FAILED_SPACE_REQUEST_LIMITS = 3;
> > +
> > +  public static final boolean DEFAULT_USER_DISABLE_LOCAL_READ = false;
>
> config param is usually about enable xxx, how about:
> DEFAULT_USER_ENABLE_LOCAL_READ = true?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amplab/tachyon/pull/868/files#r29463025>.
>